### ISSUE TITLE
Query Bug

### ISSUE THREAD

**Issue by Cosido** [plan-player-analytics/Plan#1017]
Describe the bug

https://ghostbin.com/paste/4935n
This spams console on server startup, webserver also stops at analysis, everything else seems to function aside from things contained within that

Server information

Waterfall 280
MC 1.12.2
Forge 2809
SpongeForge 3616
Plan 4.8.1-b3

**Comment by Cosido** [comment]
https://ghostbin.com/paste/486dm
https://www.spigotmc.org/resources/anti-vpn-bungee.58716/

Also getting this on startup, unsure if its related to that or not

**Comment by Rsl1122** [comment]
The second error is unrelated, but I don't know enough context to say if it's a Plan issue or AntiVPN issue.
To know that the enable order of these 2 plugins would have to be known (Does Plan fail to enable, does Plan enable before AntiVPN) - If you would like to discuss the second error more, please open a separate issue for it.

**Comment by Cosido** [comment]
The second I dont care about, the plugins seem to function, So no worries, just a tiny bit of spam on proxy restart, can deal with it. 

The query/anaylsis problem however is a bit more pressing. 
https://ghostbin.com/paste/4935n

That bin is from my vanilla server, when I try to generate a anaylsis report on the webserver. I run a Proxy/Hub/Build/Vanilla/Pixelmon/FTBInteractions server. Its showing data for all servers, but only generates an analysis for the Hub/Build (Which have minimum mods, if that matters), the rest stop on generate, and spame the consoles. I havent added anything I believe would interfere, I did update the the b3 DEV build. But I did that for all servers, and two were not affected clearly, Ive also since downgraded all to 4.8.1 and get the same problem, nothing has changed with the database/webserver/etc

Providing this extra info in case it helps!
